frontend/src/metabase/dashboard/components/DashboardGrid.tsx

```diff
@@ -175,17 +186,31 @@ class DashboardGridInner extends Component<
       props.dashcardData,
     );
 
+    const initialCardSizes = getInitialCardSizes(
+      props.dashboard.dashcards,
+      this.state?.initialCardSizes,
```

Comment:
Has to be optional chained otherwise it cause errors. This is also the case in the original diff https://github.com/metabase/metabase/pull/53213/files#diff-156bd6482f18fc7a624b700eae3c05243a2c7965a14a84b423f7fe2e25d2618fL320-L328 - let's keep it the same as before